docs/devenv.rst

```diff
@@ -206,7 +206,6 @@ includes are defined in ``setup.cfg``.
 To upgrade the development tools in the future, just rerun the exact same
 command.
 
-
```

Comment:
Did you mean to remove the line?

Reply:
I can put it back, just want to check before I do....

Reply:
please, that was a mistake.